app/src/main/java/org/fossasia/openevent/app/data/auth/AuthServiceImpl.java

```diff
@@ -128,4 +131,41 @@ public Completable changePassword(ChangePassword changePassword) {
             .subscribeOn(Schedulers.io())
             .observeOn(AndroidSchedulers.mainThread());
     }
+
+    @Override
+    public Observable<User> getUser(boolean reload) {
```

Comment:
This was already above as getOrganizer I guess

Reply:
I think getOrganizer is in EventRepository. If so, move all these methods and that to `userApi` and `userRepository`